data/brands/shop/mobile_phone.json

```diff
@@ -981,14 +981,14 @@
       }
     },
     {
-      "displayName": "Wind",
+      "displayName": "Wind Tre",
       "id": "wind-c4d7ba",
       "locationSet": {"include": ["it"]},
       "tags": {
-        "brand": "Wind",
-        "brand:wikidata": "Q650467",
-        "brand:wikipedia": "en:Wind (Italy)",
-        "name": "Wind",
+        "brand": "Wind Tre",
```

Comment:
When stores re-brand, we try to match the old name in an `alt_name` tag or `matchNames` property.  

I'm not sure how completely this rebrand has happened in Italy?  Are all stores now signed as "Wind Tre" or is it still a mix?